### ISSUE TITLE
is there a 'bleeding-edge' container?

### ISSUE THREAD

**Issue by bbolker** [rocker-org/website#113]
I'm sorry if this is supposed to be obvious but ...

I want to test UBSAN/ASAN on a very new devel version of R. AFAICT `rocker/r-devel-san` is running R 4.4.0/SVN release 86474, from April 2024. Should it be updated to run R 4.5.0?

If I were left to my own devices I guess I would try to grab the Dockerfiles for r-devel and r-devel-san and build my own images, but that feels like it could take me down a rabbit hole ...

If there is a better venue for clueless questions, please let me know.

**Comment by eitsupi** [comment]
You may want to see <https://github.com/r-hub/containers>

@eddelbuettel The rhub containers that have recently been offered are also used in rhub checks and I wonder if we could end up offering similar containers from the Rocker project in favor of these (I don't understand enough about these differences to know if it is worth continuing to offer them).

**Comment by bbolker** [comment]
running `docker pull ghcr.io/r-hub/containers/clang-asan:latest` now, fingers crossed ...

**Comment by eddelbuettel** [comment]
@bolker r-devel-san builds weekly, the 'devel'  in there means we use R 4.5.0 now.  See [its action schedule](https://github.com/rocker-org/r-devel-san/actions).  I can never remember if this one rebuild Monday or Tuesday night but I think it will rebuild within a few hours. Ditto for r-devel-san-clang, see [its action schedule](https://github.com/rocker-org/r-devel-san-clang/actions).  I got one of those love notes from CRAN against one of my packages too and was waiting for the weekly rebuild (though the pointed-out svn revision r86629 was over a week ago on Sunday May 26 to last week's builds should be good enough too).

@eitsupi I am truly sorry but can you rephrase what you are asking me?  rhub and rocker are two distinct services.
